packages/core-utils/src/time.js

```diff
@@ -48,12 +47,9 @@ export function formatDuration(seconds) {
  * @returns {string} formatted text representation
  */
 export function formatDurationWithSeconds(seconds) {
-  const dur = moment.duration(seconds, "seconds");
-  let text = "";
-  if (dur.hours() > 0) text += `${dur.hours()} hr, `;
-  if (dur.minutes() > 0) text += `${dur.minutes()} min, `;
-  text += `${dur.seconds()} sec`;
-  return text;
+  return formatDistance(0, 1000 * seconds, {
+    includeSeconds: true
+  });
```

Comment:
Same comment as https://github.com/opentripplanner/otp-ui/pull/271/files#r681324500.

Reply:
fixed in 55e76870 by migrating to [formatDistanceStrict](https://date-fns.org/v2.23.0/docs/formatDistanceStrict)